tests/raw_transfers.rs

```diff
@@ -102,5 +102,6 @@ fn raw_transfers_independent_alloy() {
         },
         None,
         true,
```

Comment:
After removing `must_succeed` we can keep only one `false` for everything below from here.